### ISSUE TITLE
pre with language problem in MarkdownV2 

### ISSUE THREAD

**Issue by daniyalmaroufi** [LucaDevelop/telegram-entities-decoder#4]
There is a problem with decoding in MarkdownV2 format when the sent message contains language.

Sent message:
"""
```python
def hey():
    print('hey')
```
"""

it decodes it to:

```pythondef hey():
    print('hey')
```

While it decodes the message in HTML format as:
```
<pre><code class="language-python">def hey():
    print('hey')</code></pre>
```

**Comment by LucaDevelop** [comment]
Hello there,
thanks for submitting your issue. I'll check it out as soon as I can.

**Comment by LucaDevelop** [comment]
Fixed with release v1.3.1
